pkg/instrumentation/upgrade/upgrade_suite_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package upgrade
```

Comment:
Can you improve the test coverage?

Reply:
This is good enough, otherwise I would have to mock the k8s client or test the logging... 

![Untitled](https://user-images.githubusercontent.com/5618424/142610351-8c006f79-1e25-4562-ac6a-ed043f031bb0.png)